sonar-csharp-plugin/pom.xml

```diff
@@ -63,7 +63,7 @@
   </scm>
 
   <properties>
-    <sonar.version>7.3</sonar.version>
+    <sonar.version>7.9</sonar.version>
     <sonarLTS.version>6.7</sonarLTS.version>
```

Comment:
I checked with @henryju who confirmed that the `SonarLTS.version` should also be updated to 7.9 (it's used to set the minimum supported version of SonarQube, which should be the latest LTS i.e. 7.9)

Reply:
Discussed offline: a separate ticket already exists for updating the minimum version: https://github.com/SonarSource/sonar-dotnet/issues/2692